yeonjae/programmers-1/문자열내림차순으로배치하기.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+
+  return s.split("").sort().reverse().join("");
+}
```

Comment:
```suggestion
function solution(s) {
  return s.split("").sort().reverse().join("");
}
```
- 잘푸셨어요 ! 윤우님 코드보시면 reverse없이 sort만으로도 풀 수 있으니 참고해주세요 👍

Reply:
오 넵 윤우님코드 참고해보겠습니다 😁😁